bin/crowbar_batch

```diff
@@ -458,7 +458,7 @@ def export_barclamp(barclamp, alias_by_host)
     end
   end
 
-  proposals = get_proposal_list(barclamp)
+  proposals = get_proposal_list(barclamp)[barclamp]
```

Comment:
I'm not the one who changed the API ;-) What is returned now is something like `["crowbar", ["default"]]`